client/src/components/Trainings.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <styled-title>
+    Trainings
+    <styled-button>Add new training</styled-button>
+  </styled-title>
+</template>
+
+<script>
+import styledComponents from '../common/styledComponents';
```

Comment:
I would suggest to import components directly. Since the amount of components will grow - we don't want to import all of them, maybe we could:
```js
import { StyledButton, StyledTitle } from '../common/styledComponents';
```
?
This should allow webpack to import only what is needed

Reply:
Looks better agree